apps/dashboard/src/components/time-display-hover-card.tsx

```diff
@@ -8,7 +8,9 @@ interface TimeDisplayHoverCardProps {
 }
 
 export function TimeDisplayHoverCard({ date, children, className }: TimeDisplayHoverCardProps) {
-  const utcTime = format(new Date(date.toUTCString()), 'MMM d yyyy, HH:mm:ss');
+  // For UTC, we adjust the date by the local timezone offset
+  const utcDate = new Date(date?.getTime() + date?.getTimezoneOffset() * 60000);
+  const utcTime = format(utcDate, 'MMM d yyyy, HH:mm:ss');
```

Comment:
Wanted to avoid adding dates-fns-tz, so a manually timezone implementation is added

Reply:
Why don't we use the Intl API https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat?